neurodocker/templates/ndfreeze.yaml

```diff
@@ -0,0 +1,9 @@
+# Instructions to run `nd_freeze` in NeuroDebian images.
+
+generic:
+  custom:
+    dependencies:
+      apt: neurodebian-freeze
+    instructions: |
+      {{ ndfreeze.install_dependencies() }}
```

Comment:
I wonder if it could/should be optimized --  `nd_freeze` will come available pre-installed  in NeuroDebian docker base images ([very soon hopefully](https://github.com/docker-library/official-images/pull/5058)), so in principle no dedicated line to install it would be needed.
Although probably there should be no harm if a dedicated apt-get install of it would be called first... and https://github.com/neurodebian/neurodebian/pull/43 probably would be of no (negative) effect since neurodocker would clean up anyways.

Reply:
we can hold off on merging this pr until the neurodebian images are rebuilt, and i'll remove the `apt-get install`.

Reply:
I think they are now:
```shell
$> docker pull neurodebian:stretch
stretch: Pulling from library/neurodebian
bc9ab73e5b14: Already exists 
b323f9a9fdf6: Already exists 
9294e84bac26: Already exists 
474b78678a62: Already exists 
e955ce81ae0d: Pull complete 
Digest: sha256:c3eb8abe2fde32b4333b4fe9efdd698afabf4c49ba07784372710b813c802c8e
Status: Downloaded newer image for neurodebian:stretch
$> docker run -it --rm neurodebian:stretch nd_freeze
Invalid command parameters
USAGE: nd_freeze [--keep-original-apt-sources] date
```
but that means
- build would fail if ppl have older version of image
- build would not work if base image a stock debian image (which soon will get neurodebian-freeze) 
- also might be worth `apt-get` ing it just to get nd_freeze updated happen we fix things up
So may be let's leave indeed as is for now ;)?